test/integration/reporters.spec.js

```diff
@@ -62,6 +62,32 @@ describe('reporters', function () {
         done(err);
       });
     });
+
+    it('does not include ansi escape sequences (issue: 4526)', function(done) {
```

Comment:
[Naming] Nit: behaviors and tests have a habit of morphing over time...

```suggestion
    it('does not include ansi escape sequences', function(done) {
```

(context: I think there are only two examples of a test name including something like this, from 3 someone who isn't maintaining anymore, 3 years ago - if this ever was a Mocha practice, it wasn't widely adopted)